src/crimson/os/seastore/segment_cleaner.h

```diff
@@ -318,20 +330,26 @@ class SpaceTrackerI {
 
   virtual void dump_usage(segment_id_t) const = 0;
 
+  virtual double calc_utilization(segment_id_t segment) const = 0;
+
   virtual void reset() = 0;
 
   virtual ~SpaceTrackerI() = default;
 };
 using SpaceTrackerIRef = std::unique_ptr<SpaceTrackerI>;
 
 class SpaceTrackerSimple : public SpaceTrackerI {
+  struct segment_bytes_t {
+    int64_t live_bytes = 0;
+    int64_t total_bytes = 0;
```

Comment:
Why are these signed?

Reply:
I made them signed just so that I can test the validity of modifications to these two fields by checking whether they are positive.